analytics/src/test/java/com/segment/analytics/IntegrationOperationTest.java

```diff
@@ -361,4 +366,44 @@ public void eventPlanOverridesSchemaDefault() throws IOException {
                 + "}"));
     verify(integration, never()).track(payload);
   }
+
+  @Test
```

Comment:
These tests have nothing to do with destination middleware, but felt good to have as unit tests